lib/id-validator.js

```diff
@@ -58,6 +58,9 @@ function validateId(doc, refModelName, value, conditions, respond) {
 }
 
 function validateIdArray(doc, refModelName, values, conditions, respond) {
+	if (values == null) {
```

Comment:
Careful with your indentation - chances are this won't be accepted unless you fix this.